functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/testbed/TestRepository.java

```diff
@@ -3,6 +3,6 @@
 public class TestRepository {
 
     public static String testRepository() {
-        return System.getProperty("test.bed.repo", "https://github.com/arquillian/smart-testing-dogfood-repo.git");
+        return System.getProperty("test.bed.repo", "https://github.com/hemanik/smart-testing-dogfood-repo.git");
```

Comment:
I think you forget to change this

Reply:
No, this is on purpose. Will change it once the review is complete, because the dogfooding repo does not have the latest tag yet.

Reply:
the code contained inside of the tag is [here](https://github.com/hemanik/smart-testing-dogfood-repo/commit/eabc6b93f433e2cd8af0d9f0fbf3054382933ff5):
```
+    <!-- skipTests property -->
+    <skipTests>true</skipTests>			 
   </properties>
 
   <issueManagement>
@@ -79,7 +82,8 @@
     <plugins>
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
-        <configuration>
+	<configuration>
+	  <skipTests>${skipTests}</skipTests>	
```
the property shouldn't be inside of the `<properties>` tag, but only in the surefire confiugration

Reply:
as mentioned here: https://github.com/arquillian/smart-testing/pull/211/files#r142849360